test/suites/dev/moonbase/test-staking/test-parachain-bond.ts

```diff
@@ -35,11 +39,11 @@ describeSuite({
         );
```

Comment:
If these methods are considered deprecated, we should update the tests to use the new `setInflationDistributionConfig` method

Reply:
Not a blocker tho

Reply:
We can migrate it when removing the deprecated methods